nixos/modules/services/networking/tayga.nix

```diff
@@ -103,13 +105,20 @@ in
       dataDir = mkOption {
         type = types.path;
         default = "/var/lib/tayga";
-        description = "Directory for persistent data";
+        description = "Directory for persistent data.";
```

Comment:
I'm unsure about the trailing dots, any specific reason you added them?

Reply:
Just to make it consistent in this module.